production/readOnly-openAPI.json

```diff
@@ -13057,7 +13210,7 @@
           "holder_id": {
             "type": "string",
             "example": "cust_eWgXlBBiY6THFRJwX45Iakv4",
-            "description": "Unique customer ID of the campaign owner."
+            "description": "Unique customer identifier of the redeemable holder."
```

Comment:
source id is also an "identifier"... imo
only example suggest that we are talking about voucherify id instead of source id.